tests/src/test_cli.py

```diff
@@ -39,6 +39,19 @@ def test_module_doesnt_exist(runner: CliRunner):
     )
 
 
+def test_python_file_not_in_sys_path(runner: CliRunner, tmpdir):
+    file = tmpdir / "foo.py"
```

Comment:
It looks like `tmp_path` is now recommended over `tmpdir`: https://docs.pytest.org/en/latest/how-to/tmp_path.html

It returns `pathlib.Path` instead of the legacy `py.path.local`.

Reply:
Didn't know about this! Excellent! love `pathlib`